src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -518,14 +537,15 @@ void Convolution::getSupportedDescriptors() {
                 // for input of inplace tensor precision) to FP32. This will add reorder for that in-place tensor
                 // bofore the fused convolution. This behaviour might be more correct regarding expected markup
                 // of the graph but performance of first and second approaches might be different. Need to verify
-                outputDataType = eltwisePrecision == Precision::BF16 ? memory::data_type::bf16 : memory::data_type::f32;
+                outputDataType = getSupportedDataType(eltwisePrecision);
```

Comment:
Yes, I did it on purpose, because 5D depthwise convolution on bf16/f16 have performance issue, and we fallback to f32 in general (when no EltwiseAdd fused), and we should keep this behavior when there is EltwiseAdd fused.